Makefile

```diff
@@ -71,7 +71,7 @@ docker: proxy-docker
 
 .PHONY: proxy-docker
 proxy-docker:
-	docker build -t gomods/athens -f cmd/proxy/Dockerfile .
+	docker build -t gomods/athens -f cmd/proxy/Dockerfile --build-arg GOLANG_VERSION=${GOLANG_VERSION} .
```

Comment:
@davidpfarrell would you be willing to update the [`GOLANG_VERSION`](https://github.com/gomods/athens/blob/2792796172b63628718b0c57f60e53b206497f98/Makefile#L5) Makefile variable to 1.13 as well?

Also, can you add a really short comment explaining what this new build arg does?

Something like "you might want to change the Go version Athens uses on the background because of $REASONS"

Reply:
Greetings!

> would you be willing to update the GOLANG_VERSION Makefile variable to 1.13 as well?
Done!

> can you add a really short comment explaining what this new build arg does?
Done!